apps/prairielearn/elements/pl-drawing/defaults.py

```diff
@@ -44,6 +45,7 @@
 element_defaults: ElementDefaults = {
     "gradable": False,
     "answers-name": "",
+    "aria-label": None,
```

Comment:
We might want to differentiate between "label" (typically a short title) and "description" (a longer and more detailed text alternative). For instance, when using the VoiceOver rotor to navigate through images on a page, the label is used to announce the image; it's generally better that that be something short. If the user wants detailed information about the image, they can find that after navigating to the image.

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-description

Reply:
Changed in https://github.com/PrairieLearn/PrairieLearn/pull/11867/commits/2aaedffecb7da9723b5ee7184ca98bc2c66e942d

Reply:
Did you actually read that link?

> Note: aria-description is still in W3C Editor's Draft for ARIA 1.3. For the time being, continue to use [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-describedby), which has been supported since ARIA 1.1.

Reply:
And to be clear: I think we want both labels and descriptions here.

Reply:
Right, but it has 91% browser support. https://caniuse.com/?search=aria-description

Reply:
Let me double check that it works though.

Reply:
Works on NVDA.

Reply:
Are users going to be able to provide a useful aria-label here?

Reply:
> Right, but it has 91% browser support.

Great point! Seems like Safari and Firefox were the latest to gain support, and even then they've had it for over a year now. That makes me comfortable enough to use it.

> Are users going to be able to provide a useful aria-label here?

I'd think so. In the case of the question that we changed here, "Diagram of T-shaped lifting mechanism on an incline." would be the label, and everything else would be the description.

Reply:
Changed in `1cb0b58`

Reply:
![image](https://github.com/user-attachments/assets/b23d8454-ee82-4104-b822-032758f34430)